committeeoversightapp/static/css/custom.css

```diff
@@ -102,6 +112,14 @@ h4 {
   color: black;
 }
 
+.btn-primary {
```

Comment:
You may need to set the border color, too.

<img width="1550" alt="Screen Shot 2019-10-16 at 11 14 03 AM" src="https://user-images.githubusercontent.com/12176173/66938040-14be5a00-f006-11e9-86a3-1a5f9e759fc5.png">